app/src/main/java/com/arygm/quickfix/ui/profile/ProfileScreen.kt

```diff
@@ -34,269 +41,265 @@ import androidx.compose.runtime.collectAsState
+  val email = loggedInAccount?.email ?: "Loading..."
+  val wallet = loggedInProfile?.wallet ?: "Loading..."
 
   Scaffold(
-      containerColor = colorScheme.background,
+      containerColor = MaterialTheme.colorScheme.background,
       topBar = {
         TopAppBar(
             title = {
-              Column(modifier = Modifier.fillMaxWidth().padding(8.dp).testTag("ProfileTopAppBar")) {
-                // "Profile" Title
-                Text(
-                    text = "Profile",
-                    color = colorScheme.primary,
-                    style = MaterialTheme.typography.headlineLarge,
-                    modifier = Modifier.padding(bottom = 8.dp).testTag("ProfileTitle"))
-
-                // Profile Card
-                Card(
-                    modifier =
-                        Modifier.fillMaxWidth(0.85f)
-                            .align(Alignment.CenterHorizontally)
-                            .testTag("ProfileCard"),
-                    shape = RoundedCornerShape(16.dp),
-                    colors =
-                        CardDefaults.cardColors(
-                            containerColor = colorScheme.surface,
-                            contentColor = colorScheme.onSurface),
-                    elevation = CardDefaults.cardElevation(4.dp)) {
-                      Row(
-                          verticalAlignment = Alignment.CenterVertically,
-                          modifier = Modifier.padding(7.dp)) {
-                            Icon(
-                                painter = painterResource(R.drawable.profilevector),
-                                contentDescription = "Profile Icon",
-                                tint = colorScheme.primary,
-                                modifier = Modifier.size(24.dp))
-                            Spacer(modifier = Modifier.width(65.dp))
-
-                            val displayName =
-                                if (loggedInProfile != null) {
-                                  capitalizeName(
-                                      loggedInProfile?.firstName, loggedInProfile?.lastName)
-                                } else {
-                                  "Loading..." // Isn't supposed to happen
-                                }
-                            Text(
-                                text = displayName,
-                                style = MaterialTheme.typography.bodyLarge,
-                                color = colorScheme.onBackground,
-                                modifier = Modifier.testTag("ProfileName"))
-                          }
-                    }
-              }
+              Column(
+                  modifier =
+                      Modifier.fillMaxWidth()
+                          .padding(horizontal = 16.dp)
+                          .testTag("ProfileTopAppBar"),
+                  horizontalAlignment = Alignment.Start) {
+                    Spacer(modifier = Modifier.height(16.dp))
```

Comment:
You need to refactor this to account for relative positioning. As discussed Friday with the team, a maximum of `8.dp` of horizontal padding is acceptable when coupled with the use of `Modifier.fillMaxWidth` otherwise it becomes very unreliable with varying display sizes. Consider the use of `BoxWithConstraints` to achieve the same style while still accounting for screen size. 

Check out my  [refactoring PRs](https://github.com/arygm/QuickFix/pull/149) if need be for examples on how to do so.

Reply:
I will do that :)